--- dwarf/dwarfdump/Makefile.in.orig	2008-12-16 10:20:50.000000000 -0800
+++ dwarf/dwarfdump/Makefile.in	2008-12-16 10:23:31.000000000 -0800
@@ -73,7 +73,11 @@
 #	 $(CC) $(CFLAGS) -c at_list.i -o $@
 
 _tag_tree_table.c: $(srcdir)/tag_tree.list $(srcdir)/tag_tree.c $(DIRINC)/dwarf.h
-	$(CC) $(CFLAGS)  $(LDFLAGS) -o tag_tree_build $(srcdir)/tag_tree.c
+	#
+	# always build fat.  We'll almost always have a cross-compiler
+	# available, and this just keeps it simpler.
+	#
+	$(CC) -arch ppc -arch i386 $(CFLAGS)  $(LDFLAGS) -o tag_tree_build $(srcdir)/tag_tree.c
 	# gcc -E tag_tree.list does not work, so use a .c name
 	rm -f _tmp1.c && ln -s $(srcdir)/tag_tree.list _tmp1.c
 	$(CC) $(CFLAGS) -E _tmp1.c \
@@ -85,7 +89,7 @@
 	$(CC) $(CFLAGS) -c _tag_tree_table.c -o $@ 
 
 _tag_attr_table.c: $(srcdir)/tag_attr.list $(srcdir)/tag_attr.c $(DIRINC)/dwarf.h
-	$(CC) $(CFLAGS)  $(LDFLAGS) -o tag_attr_build $(srcdir)/tag_attr.c
+	$(CC) -arch ppc -arch i386 $(CFLAGS)  $(LDFLAGS) -o tag_attr_build $(srcdir)/tag_attr.c
 	# gcc -E tag_attr.list does not work, so use a .c name
 	rm -f _tmp2.c && ln -s $(srcdir)/tag_attr.list _tmp2.c
 	$(CC) $(CFLAGS)  -E _tmp2.c \
